sdk/eventhub/azure-eventhub/azure/eventhub/_connection_string_parser.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+try:
+    from urllib.parse import urlparse
+except ImportError:
+    from urlparse import urlparse  # type: ignore
+
+from ._common import DictMixin
+
+
+class EventHubConnectionStringProperties(DictMixin):
+    """
+    Properties of a connection string.
+    """
+
+    def __init__(self, **kwargs):
+        self._fully_qualified_namespace = kwargs.pop("fully_qualified_namespace", None)
+        self._endpoint = kwargs.pop("endpoint", None)
+        self._event_hub_name = kwargs.pop("event_hub_name", None)
```

Comment:
I think we use the term "eventhub_name" (no underscore before `event` and `hub`) in our sdk instead of "event_hub_name".